src/ray/common/ray_config_def.h

```diff
@@ -35,7 +35,7 @@ RAY_CONFIG(bool, event_stats_metrics, false)
 RAY_CONFIG(bool, legacy_scheduler_warnings, false)
 
 /// Whether to enable cluster authentication.
-RAY_CONFIG(bool, enable_cluster_auth, false)
+RAY_CONFIG(bool, enable_cluster_auth, true)
```

Comment:
is this relevant?

Reply:
Will remove in the next revision, base branch was not master I guess